language/move-vm/transactional-tests/tests/control_flow/return_in_if_branch_taken_no_else.mvir

```diff
@@ -1,19 +0,0 @@
-//# publish
```

Comment:
I put it back. (Not sure why GitHub doesn't mark this comment as outdated, though. Maybe it's taking a while to sync since there are so many files involved now, but the line deletion you're commenting on is no longer a line deletion.)